File: TestAutomation/pages/AddPostPage.cs
using OpenQA.Selenium;

namespace TestAutomation.pages
{
    internal class AddPostPage : Page
    {
        public AddPostPage(IWebDriver driver) : base(driver)
        {
        }

        internal void addNewPost(string title, string content)
        {
            clearAndWriteText(By.Id("title"), title);
            clearAndWriteText(By.Id("content"), content);
        }

        internal EditPostPage publishPost()
        {
            waitForElementClickable(By.XPath("//*[@id='sample-permalink']/a"), 5);
            click(By.Id("publish"));
            return new EditPostPage(driver);
        }
    }
}

File: src/test/java/rst/wordpress/AddPostPage.java
package rst.wordpress;

import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;

/**
 * Created by Administrator on 2016-09-23.
 */
public class AddPostPage extends Page{
    public AddPostPage(WebDriver driver) {
        super(driver);
    }

    public void addPost(String title, String content) {
        insertText(By.xpath("/*//*[@id=\"title\"]"),title);
        insertText(By.xpath("/*//*[@id=\"content\"]"),content);
        clickElement(By.xpath("/*//*[@id=\"in-category-1680071\"]")); //kategoria

    }

    protected void insertText(By xpath, String content) {
        driver.findElement(xpath).sendKeys(content);
    }

    protected void publish() {
        clickElement(By.xpath("/*//*[@id=\"publish\"]"));

    }
}
